interfaces.go

```diff
@@ -0,0 +1,28 @@
+package ccloud
+
+import (
+	"context"
+)
+
+// Billing service allows getting billing information for an org in Confluent Cloud
+type Billing interface {
```

Comment:
Are we including this `interface` to generate mocks?

Reply:
yes this is needed